photonField.py

```diff
@@ -276,25 +277,95 @@ def getEmax(self, z=0):
         """Maximum effective photon energy in [J]"""
         return 2E-6 * eV # 0.825e-6 * eV
 
+class URB_Fixsen11:
+    """
+    Universal Radio Background as measured by ARCADE2.
+    Note that the frequency range in this reference is more narrow than for other models.
+    Therefore, this should be used carefully.
+
+    Reference:
+      D. J. Fixsen et al.
+      The Astrophysical Journal 734 (2011) 5.
+      https://arxiv.org/abs/0901.0555
+    """
+    name = 'URB_Fixsen11'
+    info = 'URB_Fixsen11'
+    redshift = None
 
-class CRB_ARCADE2:
-    def getDensity(self, eps):
+    def getDensity(self, eps, z = 0.):
         """
         Spectral number density dn/deps [1/m^3/J] at z = 0.
         """
-        # T = 1.26 +- 0.09 K (nu/GHz)^-2.6 +- 0.04, see Holder 2012
-        T = 1.26 * (nu/1e9)**-2.6
-        return 8*np.pi / c0**3 / h**3 * eps**2 / (np.exp(eps/(kB*T)) - 1)
+        eps = np.r_[eps]
+        nu = eps / h
+        T = T_CMB + 24.1 * np.power(nu / 3.1e8, -2.6)
+        I = 8. * np.pi / c0 ** 3 / h ** 3 * eps ** 2 / (np.exp(eps / (kB * T)) - 1.)
+        I[eps < self.getEmin()] = 0.
+        I[eps > self.getEmax()] = 0.
+        return I
+
+    def getEmin(self, z = 0.):
+        """Minimum effective photon energy in [J]"""
+        return 2.2e6 * h
```

Comment:
Is this really correct; units of h = m^2 * kg / s=J*s != J ?

Reply:
2.2e6 Hz is the frequency, which goes up to ~10 GHz (figure 5 of the paper). Therefore, the units are correct.